bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -0,0 +1,177 @@
+          {{- else }}
+          value: {{ .Values.externalDatabase.host | quote }}
+          {{- end }}
+        - name: AIRFLOW_DATABASE_PORT_NUMBER
+          {{- if .Values.postgresql.enabled }}
+          value: "5432"
+          {{- else }}
+          value: {{ .Values.externalDatabase.port | quote }}
+          {{- end }}
+        - name: REDIS_HOST
+          {{- if .Values.redis.enabled }}
+          value: {{ template "airflow.redis.fullname" . }}
+          {{- else }}
+          value: {{ .Values.externalRedis.host | quote }}
+          {{- end }}
+        - name: REDIS_PORT_NUMBER
+          {{- if .Values.redis.enabled }}
+          value: "6379"
+          {{- else }}
+          value: {{ .Values.externalRedis.port | quote }}
+          {{- end }}
+        - name: REDIS_PASSWORD
+          valueFrom:
+            secretKeyRef:
+            {{- if .Values.redis.enabled }}
+              name: {{ template "airflow.redis.secretName" . }}
+              key: redis-password
+            {{- else }}
+              name: {{ printf "%s-%s" .Release.Name "externalredis" }}
+              key: redis-password
+            {{- end }}
+        - name: AIRFLOW_EXECUTOR
+          value: "CeleryExecutor"
+        - name: AIRFLOW_FERNET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ if .Values.airflow.auth.existingSecret }}{{ .Values.airflow.auth.existingSecret }}{{ else }}{{ template "airflow.fullname" . }}{{ end }}
+              key: airflow-fernetKey
+        - name: AIRFLOW_WEBSERVER_HOST
+          value: {{ template "airflow.fullname" . }}
+        - name: AIRFLOW_LOAD_EXAMPLES
+          {{- if .Values.airflow.loadExamples }}
+          value: "yes"
+          {{- else }}
+          value: "no"
+          {{- end }}
+        {{- if .Values.airflow.extraEnvVars }}
+          {{ toYaml .Values.airflow.extraEnvVars | nindent 8 }}
+        {{- end }}
+        {{- if .Values.livenessProbe.enabled }}
+        livenessProbe:
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+        {{- end }}
+        volumeMounts:
+        {{- if or (.Files.Glob "files/dags/*.py") .Values.airflow.dagsConfigMap .Values.airflow.cloneDagFilesFromGit.enabled }}
+        - name: custom-dag-files
+          mountPath: /opt/bitnami/airflow/dags
+        {{- end }}
+        {{- if .Values.airflow.airflowConfigurationConfigMap }}
+        - name: custom-configuration-file
+          mountPath: /opt/bitnami/airflow/airflow.cfg
+          subPath: airflow.cfg
+        {{- end }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+      volumes:
+      {{- if or (.Files.Glob "files/dags/*.py") .Values.airflow.dagsConfigMap .Values.airflow.cloneDagFilesFromGit.enabled }}
+      - name: custom-dag-files
+        {{- if or (.Files.Glob "files/dags/*.py") .Values.airflow.dagsConfigMap }}
+        configMap:
+          name: {{ template "airflow.dagFilesConfigMap" . }}
```

Comment:
Can the dag files contain sensitive data? If so, it may make sense to have the option to pass dag files as secrets as well. We do this in https://github.com/helm/charts/pull/11233